apps/wing/src/util.ts

```diff
@@ -95,4 +95,22 @@ export function parseNumericString(text?: string) {
   return number;
 }
 
-export const currentPackage: { name: string, version: string, engines: { node: string } } = require("../package.json");
+/**
+ * Converts a binary string to a base64/base64url string
```

Comment:
The docstring should have the `@params` and `@returns` parts written as well. 
imo the summary doesn't need to mention the base64/base64url distinction, that can be in the `@returns` part instead